erts/doc/src/erlang.xml

```diff
@@ -9053,6 +9053,18 @@ ok
               <seealso marker="doc/system_principles:versions">
               System principles</seealso> in System Documentation.</p>
           </item>
+          <tag><marker id="system_info_otp_minor_version"/>
+            <c>otp_minor_version</c></tag>
+          <item>
+            <marker id="system_info_otp_minor_version"></marker>
+            <p>Returns a string containing the OTP minor version number of the
+              OTP release that the currently executing ERTS application
+              is part of. As an example if the OTP version is 21.2.3, 
```

Comment:
I'm not a native English speaker, but wouldn't "For example, if ..." be more correct than "As an example if ..." in this case?

Reply:
That sounds good to me. Will fix up.